packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -244,6 +245,7 @@ export default class Editor implements IEditor {
      * @returns HTML string representing current editor content
      */
     public getContent(mode: GetContentMode = GetContentMode.CleanHTML): string {
+        clearSelectedTableCells(this);

```

Comment:
Do not clear selection here. You need to do this in a plugin to handle ExtractContentWithDOM event